xml/System.Security.Cryptography.X509Certificates/CertificateRequest.xml

```diff
@@ -166,13 +214,34 @@
         <Parameter Name="serialNumber" Type="System.Byte[]" />
       </Parameters>
       <Docs>
-        <param name="issuerCertificate">To be added.</param>
-        <param name="notBefore">To be added.</param>
-        <param name="notAfter">To be added.</param>
-        <param name="serialNumber">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="issuerCertificate">An <see cref="T:System.Security.Cryptography.X509Certificates.X509Certificate2" /> instance representing the issuing Certificate Authority (CA).</param>
+        <param name="notBefore">The oldest date and time when this certificate is considered valid. Typically <see cref="P:System.DateTimeOffset.UtcNow"/>, plus or minus a few seconds.</param>
+        <param name="notAfter">The date and time when this certificate is no longer considered valid.</param>
+        <param name="serialNumber">The serial number to use for the new certificate. This value should be unique per issuer. The value is interpreted as an unsigned (big) integer in big-endian byte ordering.</param>
+        <summary>Creates a certificate using the established subject, key, and optional extensions using the specified certificate as the issuer.</summary>
+        <returns>An <see cref="T:System.Security.Cryptography.X509Certificates.X509Certificate2"/> object with the specified values. The returned object won't assert <see cref="P:System.Security.Cryptography.X509Certificates.X509Certificate2.HasPrivateKey" />.</returns>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentNullException"><paramref name="issuerCertificate" /> is <see langword="null" />.</exception>
+        <exception cref="T:System.ArgumentException">
+          The <paramref name="issuerCertificate"/> doesn't contain a private key.
+
+ -or-
+
+          The type of signing key represented by <paramref name="issuerCertificate"/> couldn't be determined.
+
+ -or-
+
+          <paramref name="notAfter"/> represents a date and time that happens before than <paramref name="notBefore"/>.
```

Comment:
Another "before than" (which, depending on how this gets displayed when I hit submit, is a forward reference)